ivy/stateful/__init__.py

```diff
@@ -1,3 +1,4 @@
+from . import backends
```

Comment:
The only concern here is that we'd import `tensorflow` on importing `ivy`, thus adding it as a dependency. Let's probably do something like create a submodule named `module.py` in all the backends and add the keras model implementation in the `tensorflow` backend. After that we can just add a `NativeModule` to all the backend `__init__.py` files so that based on the backend, that particular model would be contained in `ivy.NativeModule`. After that we can just do the same thing as we're doing now except for the namespace change
Apologies for the delayed review @hmahmood24, let me know what you think, thanks 😄

Reply:
Ah didn't think of it from that perspective. Thanks for the suggestion! 👍🏼